apps/extension/src/Approvals/ApproveSignTx/ApproveSignTx.tsx

```diff
@@ -134,11 +134,17 @@ export const ApproveSignTx: React.FC<Props> = ({ details, setDetails }) => {
                 case TxType.Withdraw:
                   const withdraw = tx as WithdrawProps;
                   return <div key={i}>Withdraw: {withdraw.validator}</div>;
-                case TxType.TransparentTransfer:
-                  const transfer = tx as TransparentTransferProps;
+                case TxType.Transfer:
+                  const transfer = tx as TransferProps;
                   return (
                     <div key={i}>
-                      Transparent Transfer: {transfer.target} {transfer.amount}
+                      Transparent Transfer:
+                      <br />
+                      {transfer.data.map(({ source, target, amount }, j) => (
```

Comment:
This is just a temporary measure to reflect what is in the data! We shouldn't use this UI